bot/Dockerfile

```diff
@@ -1,30 +1,27 @@
-FROM python:3.7@sha256:6eaf19442c358afc24834a6b17a3728a45c129de7703d8583392a138ecbdb092
-
-RUN set -ex && mkdir -p /var/app
-
-RUN apt-get update && apt-get install -y supervisor
-
-RUN mkdir -p /var/log/supervisor
-
-RUN python3 -m pip install poetry===1.1.13
-
-RUN poetry config virtualenvs.in-project true
-
```

Comment:
Don’t we still need the supervisor conf?

Reply:
Good catch, I just added that missing line and also updated supervisord.conf to redirect stderr logs to stdout. It makes it easier to debug.